src/main/java/com/blackducksoftware/integration/hub/api/scan/ScanStatusTypeEnum.java

```diff
@@ -0,0 +1,37 @@
+package com.blackducksoftware.integration.hub.api.scan;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum ScanStatusTypeEnum {
+	UNSTARTED,
+	SCANNING,
+	SAVING_SCAN_DATA,
+	SCAN_DATA_SAVE_COMPLETE,
+	REQUESTED_MATCH_JOB,
+	MATCHING,
+	BOM_VERSION_CHECK,
+	BUILDING_BOM,
+	COMPLETE,
+	CANCELLED,
+	CLONED,
+	ERROR_SCANNING,
+	ERROR_SAVING_SCAN_DATA,
+	ERROR_MATCHING,
+	ERROR_BUILDING_BOM,
+	ERROR;
+
+	private static final List<ScanStatusTypeEnum> PENDING_STATES = Arrays.asList(UNSTARTED, SCANNING, SAVING_SCAN_DATA,
```

Comment:
Can this be a set so that the lookup in the contains below is constant time rather than iterating through the list?